source/assets/stylesheets/styles/_blog_related_tile.scss

```diff
@@ -0,0 +1,78 @@
+.blog-related__header {
+  font-size: 1.8em;
+  letter-spacing: 0.02em;
+  padding: 30px 20px;
+
+  @include breakpoint($blog-container-width) {
+    box-sizing: border-box;
+    margin: 0 auto;
+    max-width: $blog-container-width;
+    padding-left: 30px;
+  }
+}
+
+.blog-related__container {
+  @include breakpoint($blog-container-width) {
+    box-sizing: border-box;
+    margin: 0 auto;
+    max-width: $blog-container-width;
+  }
+}
+
+.blog-related__tile {
+  background-color: $white;
+  box-sizing: border-box;
+  min-height: 130px;
+  padding: 20px;
+
+  &:hover {
+    background-color: darken($white, 5%);
+  }
+}
+
+.blog-related__text {
+  color: lighten($black, 5%);
+  font-weight: 100;
+  padding-top: 20px;
+}
+
+.blog-related__date {
+  color: $dark-gray;
+  display: inline;
+  font-size: 0.8em;
+}
+
+.blog-related__title {
+  color: $related-article-link;
+  font-size: 1.2em;
+  font-weight: 100;
+  line-height: 1.3;
+  text-decoration: none;
+}
+
+.blog-related__tag {
+  display: inline-block;
+
+  &:after {
+    color: $dark-gray;
+    content: "-";
+    padding-left: 3px;
+  }
+}
+
+.blog-related__tag-link {
+  background-color: lighten($light-purple, 55%);
+  border: 1px solid;
+  border-color: darken($light-gray, 5%);
+  display: inline-block;
+  padding-left: 10px;
+  padding-right: 10px;
+
+  a {
+    color: darken($medium-purple, 10%);
+    font-size: 0.6em;
+    line-height: 2em;
+    text-decoration: none;
+    text-transform: uppercase;
+  }
+}
```

Comment:
The filename should be `_blog_related.scss` following the convention for `block__element--modifier` -> `block.scss`